lib/WWW/Mechanize.pm

```diff
@@ -1608,6 +1608,24 @@ C<L<< form_name()|"$mech->form_name( $name )" >>> or
 C<L<< form_number()|"$mech->form_number($number)" >>>
 method or defaulting to the first form on the page).
 
+If the field is of type "file", its value should be an arrayref. Example:
+
+    $mech->field( $file_input, ['/tmp/file.txt'] );
+
+Value examples for "file" inputs, followed by explanation of what each
+index mean:
+
+    ['/tmp/file.txt']
+    ['/tmp/file.txt', 'filename.txt']
+    ['/tmp/file.txt', 'filename.txt', @headers]
+    ['/tmp/file.txt', 'filename.txt', Content => 'some content']
+    [undef,           'filename.txt', Content => 'content here']
+
+Index 0 is the filepath that will be read from disk. Index 1 is the
+filename which will be used in the HTTP request body; if not given,
+filepath (index 0) is used instead. If "Content => 'content here'" is
```

Comment:
Does this mean that, if index 1 is undef, the file attribute in the HTTP request will contain the absolute filepath given from index 0? Or will it use the basename?